hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/hive/HoodieCombineHiveInputFormat.java

```diff
@@ -579,10 +579,6 @@ public RecordReader getRecordReader(InputSplit split, JobConf job, Reporter repo
     protected CombineFileSplit inputSplitShim;
     private Map<Path, PartitionDesc> pathToPartitionInfo;
 
-    public CombineHiveInputSplit() throws IOException {
```

Comment:
this could have been called by serialization code for e.g?

Reply:
no, this would not be called by serialization code for e.g